lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -194,6 +194,20 @@ module.exports = [
       }
     }
   }, {
+    initialUrl: 'https://www.chromestatus.com/features',
```

Comment:
Oh wait we're already hitting this in smokehouse, can we just add the assertion there?